bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/core/mapping/expression/FieldToContractInputMappingExpressionBuilder.java

```diff
@@ -76,6 +76,17 @@ protected String getScriptText(final boolean isOnPool, final MappingOperationScr
         }
     }
 
+    private void setGroovyScriptName(final Expression scriptExpression, final BusinessObjectData data, final ContractInput contractInput, final boolean isOnPool) {
+        if (isOnPool) {
+            final String dataName = data.getName();
+            final String nameToUpperCase = dataName.length() > 1 ? dataName.substring(0, 1).toUpperCase() + dataName.substring(1) : dataName.substring(0, 1)
```

Comment:
for the second you can use dataName.toUpercase directly withotu substring as we checked just before that the length is < 1